src/components/OverviewApp/Notary/LargeRequestTable.tsx

```diff
@@ -1,136 +1,334 @@
-import { useContext, useEffect, useState } from "react"
-import DataTable from "react-data-table-component"
-import { Data } from "../../../context/Data/Index"
-import { LargeRequestData } from "../../../type"
-
-export const largeReqColumns = [
-    {
-        name: "Client",
-        selector: (row: LargeRequestData) => row.data,
-        sortable: true,
-        grow: 1.2,
-        wrap: true
-    },
-    {
-        name: "Address",
-        selector: (row: LargeRequestData) => row.address,
-        sortable: true,
-        cell: (row: LargeRequestData) => <div>{`${row.address.substring(0, 9)}...${row.address.substring(row.address.length - 9, row.address.length)}`}</div>
-    },
-    {
-        name: "Multisig",
-        selector: (row: LargeRequestData) => row.multisig,
-        sortable: true,
-        grow: 0.5
-    },
-    {
-        name: "Datacap",
-        selector: (row: LargeRequestData) => row.datacap,
-        sortable: true,
-        grow: 0.5
-    },
-    {
-        name: "Audit Trail",
-        selector: (row: LargeRequestData) => row.issue_number,
-        sortable: true,
-        grow: 0.5,
-        cell: (row: LargeRequestData) => <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href={row.url}>#{row.issue_number}</a>,
-    },
-    {
-        name: "Proposer",
-        selector: (row: LargeRequestData) => row.proposer.signerGitHandle,
-        sortable: true,
-        cell: (row: LargeRequestData) => <span >{row.proposer.signerGitHandle || "-"}</span>,
-        grow: 0.5,
-    },
-    {
-        name: "TxId",
-        selector: (row: LargeRequestData) => row.tx.id,
-        grow: 0.5,
-    },
-    {
-        name: "Approvals",
-        selector: (row: LargeRequestData) => row.approvals,
-        sortable: true,
-        grow: 0.5,
-    },
-]
-
-type LargeRequestTableProps = {
-    setSelectedLargeClientRequests: any
-    dataForLargeRequestTable: LargeRequestData[],
-    largeRequestListLoading: boolean
-    setDataForLargeRequestTable: any
-}
-
-const LargeRequestTable = ({ setSelectedLargeClientRequests, largeRequestListLoading, dataForLargeRequestTable, setDataForLargeRequestTable }: LargeRequestTableProps) => {
-
-    const context = useContext(Data)
-
-    const CANT_SIGN_MESSAGE = "You can currently only approve the allocation requests associated with the multisig organization you signed in with. Signing proposals for additional DataCap allocations will require you to sign in again";
-
-    const [searched, setSearched] = useState(false)
+import { ldnParser } from "@keyko-io/filecoin-verifier-tools";
+import { CircularProgress } from "@material-ui/core";
+import { useContext, useEffect, useState } from "react";
+import DataTable from "react-data-table-component";
+import { config } from "../../../config";
+import { Data } from "../../../context/Data/Index";
+import { LargeRequestData } from "../../../type";
+import verifierRegistry from "../../../data/verifiers-registry.json";
```

Comment:
we should fetch this notaries data from content repo

Reply:
Please open a different issue for this one

Reply:
ok , this is just something we dont want to forget, because before we ship this we want to adjust this.